ersilia/publish/inspect.py

```diff
@@ -257,21 +257,22 @@ def check_dependencies_are_valid(self):
             return Result(True, "Check passed.")
         return Result(False, details)
 
+
     def check_comptuational_performance(self):
         """
         Measure computational performance by serving the model and running predictions.
+        Includes tracking metrics using the --track flag.
 
-         Returns:
+        Returns:
             Result: A namedtuple containing a boolean success status and details of the check.
         """
         details = ""
 
         for n in (1, 10, 100):
-
             cmd = (
                 f"ersilia serve {self.model} && "
                 f"ersilia example -f my_input.csv -n {n} && "
-                "ersilia run -i my_input.csv && "
+                "ersilia run -i my_input.csv --track && "  # Added --track flag
```

Comment:
@AnushkaShreyam track is used when serving a model, so typically, you would want to do something like `ersilia serve eosxxxx --track`.